ui/imports/Themes/LightTheme.qml

```diff
@@ -18,11 +18,11 @@ Theme {
     property color green: "#4EBC60"
 
     property color background: white
-    property color border: grey
+    property color border: lightGrey
     property color textColor: black
-    property color currentUserTextColor: white
+    property color currentUserTextColor: black
```

Comment:
I think this one needs to stay `white`. It's for the chat bubble when it's the current user's text. It's white on blue